pkgs/by-name/dc/dc3dd/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchzip
+, perl538Packages
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dc3dd";
+  version = "7.3.1";
+
+  src = fetchzip {
+      url = "http://downloads.sf.net/sourceforge/dc3dd/dc3dd-${finalAttrs.version}.zip";
+      hash = "sha256-SYDoqGlsROHX1a0jJX11F+yp6CeFK+tZbYOOnScC6Ig=";
+  };
+
+  preConfigure = ''
+    chmod +x configure
+  '';
+
+  buildInputs = [ perl538Packages.LocaleGettext ];
```

Comment:
We cannot just use perl5Packages?

Reply:
dc3dd needs of [perl-locale-gettext](https://github.com/BlackArch/blackarch/blob/cbfe41345380c0b9d72b1e193da8acdd76822cb8/packages/dc3dd/PKGBUILD#L12) dependency that should correspond to `perl538Packages.LocaleGettext`.

Reply:
No other Perl version works?

Reply:
Just tested. It works by `perlPackages`.